tests/win_bindings.rs

```diff
@@ -0,0 +1,19 @@
+#![cfg(all(windows, feature = "clock", feature = "std"))]
+
+use std::fs;
+use windows_bindgen::bindgen;
+
+#[test]
+fn gen_bindings() {
+    let input = "src/offset/local/win_bindings.txt";
+    let output = "src/offset/local/win_bindings.rs";
+    let existing = fs::read_to_string(output).unwrap();
+
+    let log = bindgen(["--etc", input]).unwrap();
+    eprintln!("{}", log);
+
+    let new = fs::read_to_string(output).unwrap();
+    if !new.lines().eq(existing.lines()) {
```

Comment:
Why are we comparing this on a line-by-line basis? To get around the `\r\n` vs `\n` differences? If so, please add a comment to that effect.

Reply:
Pretty sure it's that, all files should be checked in to the repo with `\n` but because the test runs on Windows (`windows-bindgen` runs just fine on Linux, fwiw!) the output will be `\r\n`.

Personally I'd just run this test in the CI with a helpful "how to regenerate" message and a `git status --porcelain` check for files that may not be checked in, as linked above:

https://github.com/Traverse-Research/amd-ext-d3d-rs/commit/d5ba12272795a7364de43646f0d1bc1edd65bd08#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR62

(Note that this line [changed a bit on main](https://github.com/Traverse-Research/amd-ext-d3d-rs/blob/2adfbb50cf08e7836d06e927c7e1b2b2226cc182/.github/workflows/ci.yml#L60) because we're back to `cargo r -p api_gen`.)

Reply:
> (windows-bindgen runs just fine on Linux, fwiw!)

I just rebooted from Linux to Windows to work on this :rofl:

Reply:
@MarijnS95 I have to admit I am pretty bad at all this :smile:. Is it easy for you to change this PR to something better?
And thanks for helping out already!